README.md

```diff
@@ -40,47 +54,44 @@ The Crossmint SDK is a powerful collection of packages designed to simplify bloc
 - 💻 **Developer-First**: Build end-to-end blockchain solutions without deep Web3 expertise
 - 🔒 **Secure**: Enterprise-grade security for all blockchain interactions
 - 💳 **Fiat-First**: Enable users to participate in Web3 without cryptocurrency
-- 🆓 **Free for Sellers**: No costs to integrate
+- 🆓 **Free to Start**: Start developing at no cost
 
 ### Core Features
 
-- 💳 Wallet-less credit card purchases for NFTs
+- 💳 NFT checkout: with credit card and cross-chain
 - 🔐 Authentication and session management
-- 👛 Smart wallet integration
-- 📜 Verifiable credentials management
-- 🌐 Cross-platform compatibility
+- 👛 Embedded wallets
+- 📜 Verifiable credentials
 
 ## Package Overview
 
 ### Client-Side Packages
 
-- `@crossmint/client-sdk-react-ui`: React components and hooks for integrating blockchain, authentication, and NFT-related functionalities into your application.
-
-- `@crossmint/client-sdk-base`: Core client-side functionality and essential building blocks for Crossmint integration.
-
-- `@crossmint/client-sdk-auth`: Client-side authentication tools for managing user sessions and integrating various authentication methods.
-
-- `@crossmint/client-sdk-verifiable-credentials`: Comprehensive tools for managing verifiable credentials, including verification, presentation, and decryption capabilities.
-
-- `@crossmint/client-sdk-smart-wallet`: Smart wallet functionality featuring passkey support, auth providers (Privy, Dynamic), and blockchain providers (Viem account, EIP1193 compatible).
+- `@crossmint/client-sdk-react-ui`: React SDK, with UI components.
+	
+- `@crossmint/client-sdk-auth`: Headless typescript SDK for managing auth and user profiles.
+	
+- `@crossmint/client-sdk-verifiable-credentials`: Headless typescript SDK for Verifiable Credentials.
+	
+- `@crossmint/client-sdk-smart-wallet`: Headless typescript SDK for smart wallets.
+	
+Other:
+	
+- `@crossmint/client-sdk-base`: Core client-side functionality and essential building blocks for Crossmint integration. Typically you don't need to integrate against this directly.
 
 ### Server-Side Packages
 
-- `@crossmint/server-sdk`: Server-side authentication and session management tools, specifically designed for SSR applications like Next.js.
-
-### Common Packages
-
-- `@crossmint/common-sdk-base`: Shared utilities, enums, and common functionality across the SDK ecosystem.
-
-- `@crossmint/common-sdk-auth`: Shared authentication utilities and tools used across different SDK packages.
+- `@crossmint/server-sdk`: Server-side SDK.
 
 ### Demo Applications
 
-- `@crossmint/client-sdk-react-ui`: A NextJS application showcasing the full capabilities of the SDK, including authentication and smart wallet integration.
+- `🔒 Smart Wallet + Auth Demo`: A NextJS application showcasing the full capabilities of the SDK, including authentication and smart wallet integration.
+
+Link to [Smart Wallets Demo (Next.js Starter Kit)](./apps/wallets/smart-wallet/next/README.md)
```

Comment:
I meant the bullet point should have the link inside it, not to put it separately like here